schemas/jsonschema/DataField.json

```diff
@@ -1,7 +1,7 @@
 {
   "$comment": "Generated from DataField by @foxglove/message-schemas",
   "title": "DataField",
-  "description": "List of fields included for every entity in an accompanying `data` field array",
+  "description": "List of fields for every element in an accompanying `data` field array",
```

Comment:
DataField is now PackedElementField?

Reply:
The `data` field part of this should be removed from this description cause thats a property of the message. The message doesn't have to use the `data` name for the array field.

Reply:
The rename is a separate PR which I requested your review on